tests/glm/test_glm_cv.py

```diff
@@ -37,7 +36,7 @@ def test_normal_elastic_net_comparison(l1_ratio, fit_intercept, convert_x_fn):
     tol = 1e-9
 
     n_predict = 10
-    X, y, _ = make_regression(
+    X, y, _ = skl.datasets.make_regression(
```

Comment:
On the other hand I like using the qualified classname here as `make_regression` sounds like it could come from anywhere.